### PR TITLE
Update Project Slider

### PR DESCRIPTION
**Pull request by Joeedwin073**

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -13,12 +14,17 @@ class HomeController extends Controller
      * @return \Illuminate\Http\Response
      */
     public function index()
-    {
+    {   
+        $recent_projects = Expense::orderBy('created_at','DESC')
+                ->skip(0)
+                ->take(2)
```
Comment (goodhands) [inline app/Http/Controllers/HomeController.php:20]: The task specification states that 4 projects should be displayed not 2

File: resources/views/pages/home.blade.php [hunk 2]
```diff
@@ -29,27 +29,18 @@
             <img src="{{asset('images/angle-right.svg')}}" alt="" class="arrow-right2">
           </div>
           <div class="gallery">
-            <div class="card1 card">
+          @foreach($recent_projects as $recent_project)
+            <div class="card{{$loop->iteration}} card">
```
Comment (goodhands) [inline resources/views/pages/home.blade.php:33]: Where is `$loop` defined?

File: resources/views/pages/home.blade.php [hunk 3]
```diff
@@ -29,27 +29,18 @@
...
                 {{-- <p class="tag">New</p> --}}
               <div class="project">
-                  <p>Contruction of Lagos-Ibadan Express road <br>
-                    Ministry of Power, Works and Housing</p>
+                  <p>{{$recent_project->project}}<br></p>
                   <div class="d-flex justify-content-between mt-4 align-items-center">
                       <p>Cost of Project: </p>
-                      <p id="cost">&#8358;20bn</p>
+                      <p id="cost">{{$recent_project->amount_spent}}</p>
```
Comment (goodhands) [inline resources/views/pages/home.blade.php:39]: This should be converted to proper money format with the Naira currency prepended. I believe there is an `amount()` helper method on `App\Expense.php`

**Review by goodhands — CHANGES_REQUESTED** [review]
Make these changes for now, I will test locally and give more feedback

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (Joeedwin073) [reply app/Http/Controllers/HomeController.php:20]: Alright sir.  Done

**Pull request closed without merging**